tensorrt_llm/_torch/distributed/ops.py

```
@@ -21,56 +20,48 @@
 
 def get_allreduce_mnnvl_workspace(
     mapping: Mapping, dtype: torch.dtype
 ) -> Tuple[McastGPUBuffer, torch.Tensor, torch.Tensor, int]:
-    if not hasattr(_thread_local,
-                   f'allreduce_mnnvl_workspaces_{mapping.pp_rank}'):
-        setattr(_thread_local, f'allreduce_mnnvl_workspaces_{mapping.pp_rank}',
-                {})
+    if not hasattr(_thread_local, f"allreduce_mnnvl_workspaces_{mapping.pp_rank}"):
+        setattr(_thread_local, f"allreduce_mnnvl_workspaces_{mapping.pp_rank}", {})
 
     force_mn = os.environ.get("TRTLLM_FORCE_MNNVL_AR", "0") == "1"
```

Ruff (0.12.2)

Local variable `force_mn` is assigned to but never used

Remove assignment to unused variable `force_mn`

(F841)